jbpm-case-mgmt/jbpm-case-mgmt-impl/src/test/java/org/jbpm/casemgmt/impl/CaseRuntimeDataServiceImplTest.java

```diff
@@ -212,6 +215,16 @@ public void testUserTasksInCase() {
         }
     }
 
+    @Test
+    public void testUserTasksInCaseAsPotentialOwners() {
+        assertTasksInCaseAsPotentialOwner("ActorId", new UserImpl("john"));
+    }
+
+    @Test
+    public void testGroupTasksInCaseAsPotentialOwners() {
+        assertTasksInCaseAsPotentialOwner("GroupId", new GroupImpl("HR"));
```

Comment:
Same here, if you added
```suggestion
        assertTasksInCaseAsPotentialOwner(GROUP_ID, new GroupImpl("HR"));
```

Reply:
Done